config/locales/en.yml

```diff
@@ -340,12 +340,10 @@ en:
     sign_up_as_coach: "Join us as a coach"
     code_of_conduct: "code of conduct"
     new:
-      intro: "We offer a welcoming, inclusive and learning-friendly environment, and have a zero tolerance policy towards any form of harassment or inappropriate behavior. Before you sign up, read our"
+      intro_html: "We provide a welcoming, inclusive, and supportive learning environment, with a strict zero-tolerance policy for any form of harassment or inappropriate behavior. Please take a moment to review our <a href='https://codebar.io/code-of-conduct'>code of conduct</a> before signing up."
```

Comment:
Instead of hard coding the href of the links like this can we pass it in as a variable to the translation string? That way we can make use of the path helpers (where they are available) and if the path every changes we'll only have to update it in one place. https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations